airbyte-webapp/src/components/connectorBuilder/YamlEditor/YamlEditor.module.scss

```diff
@@ -5,17 +5,22 @@
   display: flex;
   flex-flow: column;
   height: 100%;
+  position: relative;
 }
 
 .control {
   flex: 0 0 auto;
-  background-color: transparent;
+  background-color: colors.$grey-30;
   display: flex;
   padding: variables.$spacing-md;
   flex-direction: row;
-  justify-content: flex-end;
 }
 
 .editorContainer {
+  padding-top: variables.$spacing-xl;
   flex: 1 1 0;
 }
+
+.downloadButton {
```

Comment:
You don't need this, you can simply give the `.control` class a `flex-direction: row-reverse` instead of row so it starts from the right

Reply:
Done!